### PR TITLE
Included .anki2 Import Instructions

### PR DESCRIPTION
**Pull request by david-allison-1**
I assume a lot of our import errors are due to anki2 imports failing.

We can direct users here to find a better solution or cut down on support work.

Related: https://github.com/ankidroid/Anki-Android/pull/6258

### CODE REVIEW

File: manual.asc [hunk 1]
```diff
@@ -513,6 +513,21 @@ You can also manually import .apkg files as follows:
  * Choose the apkg file you just copied to your device when prompted
  * Tap OK
  
+== Importing Anki Databases (.anki2)
+ 
+AnkiDroid does not yet support directly importing Anki database (`.anki2`) files. A database import will replace your collection with the contents the provided file. If you want to perform a full replacement, you should ideally import an exported `collection[.apkg/.colpkg]`, as this also includes media data.
```
Comment (mikehardy) [inline manual.asc:518]: I don't see us ever supporting this, re-wording to reflect

```suggestion
AnkiDroid does not support directly importing Anki database (`.anki2`) files. A database import will replace your collection with the contents the provided file. If you want to perform a full replacement, you should import a `collection[.apkg/.colpkg]` created in the app with the export function.
```

File: manual.asc [hunk 2]
```diff
@@ -513,6 +513,21 @@ You can also manually import .apkg files as follows:
...
+ 
+=== Importing anki2 files manually
+
+`.anki2` files can manually be imported:
```
Comment (mikehardy) [inline manual.asc:522]:
```suggestion
This is not officially supported, but `.anki2` files can manually be imported if needed, in cases of troubleshooting for example:
```

Comment (mikehardy) [reply manual.asc:522]: I'm fine with people shooting their foot off, I try to only use tools that allow it even, but expectations need to be clear - can't imagine trying to untangle the mess if someone does this and doesn't know what they are doing / are not okay with losing everything

File: manual.asc (hunk 1 above)
Comment (david-allison-1) [reply manual.asc:518]: I'd like to (not a priority - might never be) - an `anki2` file import is effectively the same as a `collection.apkg` import without media. We've already got the code to do it, and it makes for a slightly nicer user experience.

**Comment by david-allison-1** [comment]
Done. Please squash and merge as there's too many commits here.

File: manual.asc (hunk 1 above)
Comment (mikehardy) [reply manual.asc]: Whereas I really don't want to. Having multiple ways to import/export, one of which is getting into internals, violates the packaging we strive for and removes the possibility that on import/export we can do things like check if they have scheduler compatibility, or perhaps alter things for coming libanki changes etc etc. I do not want to go anywhere near it - I want to preserve the packages as they exist for import export

**Pull request merged**